cmd/rpc-go-server/fixtures/todo_types_gen.go.fix

```diff
@@ -0,0 +1,94 @@
+// Do not edit, this file was generated by github.com/apex/rpc.
+
+package server
+
+import (
+  "context"
+  "net/http"
+
+  "github.com/apex/rpc"
+  "github.com/apex/log"
+
+  "github.com/apex/rpc/cmd/rpc-go-server/api"
+)
+
+// ServeHTTP implementation.
+func (s *Server) ServeHTTP(w http.ResponseWriter, r *http.Request) {
```

Comment:
I haven't used them in a while but I think Go special-cases `testdata` directories, all the tooling avoids them I believe, we could do `./testdata/todo_types_gen.go` I think, not a big deal either way

Reply:
Ahh, didn't know this. Will keep this in mind in the future.